src/main/kotlin/no/nav/sosialhjelp/avtaler/Configuration.kt

```diff
@@ -69,6 +84,7 @@ object Configuration {
     val local: Boolean = profile == Profile.LOCAL
     val dev: Boolean = profile == Profile.DEV
     val prod: Boolean = profile == Profile.PROD
+    val baseUrl: String = this["application.baseUrl"]
```

Comment:
Er dette samme som ingress i nais.yaml? Eller baseurl for frontend?

Reply:
Det er baseurl for frontend. Trengs for redirect i browser fra digipost etter signering der.